src/App/Pages/Accounts/TwoFactorPageViewModel.cs

```diff
@@ -171,11 +180,11 @@ public void Load()
             }
             if (SelectedProviderType == null || DuoMethod)
             {
-                page.RemoveContinueButton();
+                ShowContinue = false;
```

Comment:
You can simplify this `if` block to:
`ShowContinue = !(SelectedProviderType == null || DuoMethod)`

Reply:
✅